docs/dvl/gui/dashboard.md

```diff
@@ -9,6 +9,34 @@ The default page is a dashboard which provides a summary and visualisation of bo
 
 ![](../../img/dvl_gui_dashboard.png)
 
+## Dashboard Parameters
+
+### Doppler Velocity
+Parameter                               |Description
+----------------------------------------|---------------------------------------------------------------------
+Speed                                   | Combined speed from x, y, z velocities.
+Altitude                                | Distance from seabed to DVL (unless beams blocked by something else).
+Figure of merit(See note below)         | Maximum uncertainty of measurement given in m/s.
+Ping rate                               | Rate at which data velocity is being outputted (depends on velocity).
+Velocity valid                          | Indicates if the velocity is valid to be used.
+
+!!! Note
+	Figure of merit (FOM) represent the maximum possible uncertainty represents the largest potential error in the measured speed from our DVL. This value is derived from the covariance matrix, which identifies the direction with the highest uncertainty in the sensor data. It provides a worst-case error margin. The maximum amount by which the actual speed may differ from the measured speed due to various factors from environment such as acoustic noise from other sources.
+
+    Users can use this figure of merit to assess the reliability of the speed measurements and determine whether the level of uncertainty meets their application requirements. This value allows for better decision-making and operational planning, ensuring that even in the most challenging conditions, the DVL operates within predictable error limits. 
+
+### Dead reckoning
+Parameter                               |Description
+----------------------------------------|------------------------------------------------------------------------------------------------------------------------
+Speed                                   | Speeds in x,y,z(Downward) directions with FOM that is the Euclidean norm of the standard deviations for each direction. 
+Position                                | Calculated position with FOM that is the Euclidean norm of the standard deviations for each direction. 
+Roll, pitch, Yaw                        | Orientation from the gyro.
```

Comment:
Check if yaw angle is reset on dead-reckoning reset or not.

Reply:
It is

Reply:
Right then we should emphasize this here. Something like: "Orientation from the gyro. The orientation is the change since init/reset dead reckoning and earth oriented. Ie, yaw does not point north".   Hopefully you can find a better phrasing than mine :sweat:

Reply:
Something like this:

[de6eb81](https://github.com/waterlinked/waterlinked.github.io/pull/30/commits/de6eb81597db60b5f3c2b03d4052c41bd9bedef1)